### PR TITLE
additions and edits to accommodate roles (tenant, clientcredentialsclient, user)

### PR DESCRIPTION
**Pull request by digitaljoshua**
added (for specific instances with variations of value definitions) clientcredentialclientQuery, clientcredentialclientroleSkip, clientcredentialclientroleCount

edited values for clientcredentialclientId,  clientcredentialclientCount, clientcredentialclientSkip, query-user, skip-user, count-user

### CODE REVIEW

File: content/external-references/parameters.yaml [hunk 1]
```diff
@@ -97,12 +97,24 @@ identityProviderSkip:
 clientcredentialclientId:
   type: text/plain
   value: |
-    Client credential client identifier
+    Client credential client identifier. Empty, whitespace or null identifiers will be ignored.
```
Comment (GiselleLeeOSI) [inline content/external-references/parameters.yaml:100]: Josh, this is merely a suggestion but I do notice that the parameter description especially for queries can be tightened up. In other words, some seem to be a preference of the developer who worked on the API, not a meaningful difference in the actual functionalities. Maybe you want to talk to them ask if you could come up with a handful for query paramaters.

Comment (digitaljoshua) [reply content/external-references/parameters.yaml:100]: good idea. I'm meeting with Dave O tomorrow to discuss the parameter values.

File: content/external-references/parameters.yaml [hunk 2]
```diff
@@ -34,46 +26,10 @@ clientId:
...
   value: |
     Client identifier
-id:
-  type: text/plain
-  value: |
-    Unordered list of identifiers for all clients to get. Empty or whitespace identifiers will be ignored.
-tag:
-  type: text/plain
-  value: |
-    Only return clients that have these tags
```
[the remark this answers is not in the document]
Comment (digitaljoshua) [reply content/external-references/parameters.yaml:44]: I didn't remove any parameter I did not create. those are still there. i believe the red you are seeing are from the removing in one spot and putting in another spot (the reordering of like parameters that I mention in the commit description). 
i assure you that all of these parameters you mention have not be edited. just reordered and are in the parameters.yaml file.
parametersyaml

[the remark this answers is not in the document]
Comment (digitaljoshua) [reply content/external-references/parameters.yaml:44]: here are two screen shots that capture my up to date forked parameter.yaml file
parametersscreen2
parametersscreen1

[the remark this answers is not in the document]
Comment (GiselleLeeOSI) [reply content/external-references/parameters.yaml:44]: Please check the latest commit and see liens 1, 34 and 38 for commenting in .yaml

File: content/external-references/parameters.yaml [hunk 3]
```diff
@@ -6,14 +7,40 @@ tenantId:
   type: text/plain
   value: |
     Tenant identifier
-tenantId-OCS:
+query:
```
Comment (digitaljoshua) [inline content/external-references/parameters.yaml:10]: # Standard parameters

File: content/external-references/parameters.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (digitaljoshua) [reply content/external-references/parameters.yaml:44]: good idea. i added some more in the fourth commit.

File: content/external-references/parameters.yaml [hunk 4]
```diff
@@ -134,15 +91,34 @@ dataViewId:
   type: text/plain
   value: |
     Data view identifier
-cancellation-token:
-  type: text/plain
-  value: |
-    Cancellation token
 subscriptionId:
```
Comment (GiselleLeeOSI) [inline content/external-references/parameters.yaml:94]: Could you put a comment OMF parameters before subscriptionId and topicId

File: content/external-references/parameters.yaml [hunk 5]
```diff
@@ -134,15 +91,34 @@ dataViewId:
...
+id:
+  type: text/plain
+  value: |
+    Unordered list of identifiers for all clients to get. Empty or whitespace identifiers will be ignored.
+tag:
+  type: text/plain
+  value: |
+    Only return clients that have these tags
+# Query for community 
+query-community:
```
Comment (GiselleLeeOSI) [inline content/external-references/parameters.yaml:116]: And move query-community to under community parameter

Comment (digitaljoshua) [reply content/external-references/parameters.yaml:116]: i'm assuming you mean communityId parameter, and not the tenantId-community?
will do.

File: content/external-references/parameters.yaml [hunk 6]
```diff
@@ -86,38 +70,16 @@ identityProviderTypeNameClaimId:
...
-  type: text/plain
-  value: |
-    Maximum number of roles to return
-clientcredentialclientSkip:
-  type: text/plain
-  value: |
-    Number of roles to skip
-tenantId-community:
+# Specific types of identifiers
+communityId:
```
[the remark this answers is not in the document]
Comment (digitaljoshua) [reply content/external-references/parameters.yaml:74]: ok. i'll make the change and you can approve. thanks.

**Review by GiselleLeeOSI — APPROVED** [review]
Looks good!

**Pull request merged**